src/main.c

```diff
@@ -408,6 +418,11 @@ int main(int argc, char **argv)
 			if (!recursive_scan) {
 				fts_set(ftsp, file, FTS_SKIP);
 			}
+
+			if (!scan_hidden_dirs && file->fts_info == FTS_D && file->fts_name[0] == '.' && file->fts_name[1] != '.' && file->fts_name[1] != '\0') {
```

Comment:
Can you split this into multiple lines so it's more readable?

Reply:
done